### ISSUE TITLE
Negative portal scissor size on 0.55 branch

### ISSUE THREAD

**Issue by slipher** [DaemonEngine/Daemon#1297]
On `for-0.55.0/sync` I periodically get errors about negative size from SetViewportAndScissor on map `test-portals`. This can be seen with r_checkGLErrors or the other GL debug callback thing, for example:
```
^3Warn: DEBUG_TYPE_ERROR: severity: high msg: GL_INVALID_VALUE error generated. Width and height must not be negative.
```

Some code to prevent negative values was removed on the branch by 948a3fea80d8105cbbacbc8e6d147912c277b179.

**Comment by VReaperV** [comment]
Looks like something wasn't merged properly because this was actually fixed in https://github.com/DaemonEngine/Daemon/commit/2ec4cb73e145633d3616f0d92781a7b9c24e8ffd, and the fix is there on `master`.

**Comment by VReaperV** [comment]
Fixed by #1306.
